PackageExplorer/Converters/ReferenceSetConverter.cs

```diff
@@ -19,16 +19,22 @@ public class ReferenceSetConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, _CultureInfo culture)
         {
-            var referenceSets = (ICollection<PackageReferenceSet>)value;
-            if (referenceSets.Any(d => d.TargetFramework != null))
+            if (value is ICollection<PackageReferenceSet> referenceSets)
             {
-                // if there is at least one dependeny set with non-null target framework,
-                // we show the dependencies grouped by target framework.
-                return referenceSets;
-            }
+                if (referenceSets.Any(d => d.TargetFramework != null))
+                {
+                    // if there is at least one dependeny set with non-null target framework,
+                    // we show the dependencies grouped by target framework.
+                    return referenceSets;
+                }
 
-            // otherwise, flatten the groups into a single list of dependencies
-            return referenceSets.SelectMany(d => d.References);
+                // otherwise, flatten the groups into a single list of dependencies
+                return referenceSets.SelectMany(d => d.References);
+            }
+            else
+            {
+                return new List<PackageReferenceSet>();
```

Comment:
As we are returning an enumerable, let's not allocate further with a 
```suggestion
                return Enumerable.Empty<PackageReferenceSet>();
```